packages/manager/modules/pci/src/projects/project/activate/activate.routing.js

```diff
@@ -48,6 +48,11 @@ export default /* @ngInject */ ($stateProvider) => {
 
         if (voucherAmount) {
           return claimDiscoveryVoucher(voucherPayload)
+            .catch((err) => {
+              if (!/VOUCHER_ALREADY_USED/.exec(err.data?.message)) {
+                throw err;
+              }
+            })
```

Comment:
There's already a catch block in Line 63, add the `if` there and throw the error.